sample-nginx.conf

```diff
@@ -53,6 +53,15 @@ http {
         ssl_prefer_server_ciphers       on;
         add_header                      Strict-Transport-Security max-age=15552001;
 
+        # Serve static resources directly.
+        location /assets/*

```

Comment:
Maybe there needs to be an opening bracket here.

Reply:
and I think it needs a tilde for the regex to work right? But then after it works it seems I cannot use the menu on mobile.  My phone is pretty busted up thought, so I'm not sure.

`location ~ ^/assets/*`

Reply:
Well spotted. I opened #443 to fix this.

I tested this change on my own server but I think I forgot to copy the extra bracket back into the PR - oops!

Tilde is not necessary. Tilde forces the Regex match to be case-sensitive, which we do not need here.